### PR TITLE
Update all C2PA references to point to version 2.1

### PR DESCRIPTION
**Pull request by scouten-adobe**
Add prohibition on using v1 timestamps as defined by C2PA.

### CODE REVIEW

**Comment by scouten-adobe** [comment]
Closes #164.

File: docs/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -476,7 +478,7 @@ An example in https://www.rfc-editor.org/rfc/rfc8949.html#name-diagnostic-notati
 
 === Presenting the `signer_payload` data structure for signature
 
-Prior to presenting the _<<_credential_holder,credential holder>>_ with the `signer_payload` data structure for signature, the _<<_referenced_assertions,referenced assertions>>_ MUST themselves be created. This process is described in link:++https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#_creating_assertions#_creating_assertions++[Section 10.3.1, “Creating assertions,”] of the C2PA technical specification.
+Prior to presenting the _<<_credential_holder,credential holder>>_ with the `signer_payload` data structure for signature, the _<<_referenced_assertions,referenced assertions>>_ MUST themselves be created. This process is described in link:++https://c2pa.org/specifications/specifications/2.1/specs/C2PA_Specification.html#_creating_assertions#_creating_assertions++[Section 10.3.1, “Creating assertions,”] of the C2PA technical specification.
```
Comment (lrosenthol) [inline docs/modules/ROOT/pages/index.adoc:481]:
```suggestion
Prior to presenting the _<<_credential_holder,credential holder>>_ with the `signer_payload` data structure for signature, the _<<_referenced_assertions,referenced assertions>>_ MUST themselves be created. This process is described in link:++https://c2pa.org/specifications/specifications/2.1/specs/C2PA_Specification.html#_creating_assertions++[Section 10.3.1, “Creating assertions,”] of the C2PA technical specification.
```

File: docs/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -559,30 +561,30 @@ The `pad1` and `pad2` fields of an *<<_identity_assertion,identity assertion>>*
...
 
-When using a data hash assertion, a _<<C2PA claim generator>>_ MUST follow the process described in link:++https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#_creating_a_claim++[Section 10.3, “Creating a claim,”] and link:++https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#_multiple_step_processing++[Section 10.4, “Multiple step processing,”] of the C2PA technical specification with additional steps as described below:
+When using a data hash assertion, a _<<C2PA claim generator>>_ MUST follow the process described in link:++https://c2pa.org/specifications/specifications/2.1/specs/C2PA_Specification.html#_creating_a_claim++[Section 10.3, “Creating a claim,”] and link:++https://c2pa.org/specifications/specifications/2.1/specs/C2PA_Specification.html#_multiple_step_processing++[Section 10.4, “Multiple step processing,”] of the C2PA technical specification with additional steps as described below:
```
Comment (lrosenthol) [inline docs/modules/ROOT/pages/index.adoc:575]: Multiple step processing is optional, IIRC...

File: docs/modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -612,7 +614,7 @@ sequenceDiagram
 
 === Validation method
 
-IMPORTANT: Validation of the _<<C2PA Manifest>>_ MUST be completed with a finding that the manifest is at least _well-formed_ as per link:++https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#_well_formed_manifest++[Section 14.3.2, Well-formed manifest,”] of the C2PA technical specification before a validator attempts to report on the validity of an *<<_identity_assertion,identity assertion>>.*
+IMPORTANT: Validation of the _<<C2PA Manifest>>_ MUST be completed with a finding that the manifest is at least _well-formed_ as per link:++https://c2pa.org/specifications/specifications/2.1/specs/C2PA_Specification.html#_well_formed_manifest++[Section 14.3.2, Well-formed manifest,”] of the C2PA technical specification before a validator attempts to report on the validity of an *<<_identity_assertion,identity assertion>>.*
```
Comment (lrosenthol) [inline docs/modules/ROOT/pages/index.adoc:617]: I am OK with this, but do you want to do anything more detailed using the new success/error/informational codes?

**Review by lrosenthol — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (scouten-adobe) [reply docs/modules/ROOT/pages/index.adoc:575]: Discussed with @lrosenthol just now. The requirement to use multiple-step processing is properly conditioned on using a data hash assertion. Leaving it as is.

File: docs/modules/ROOT/pages/index.adoc (hunk 3 above)
Comment (scouten-adobe) [reply docs/modules/ROOT/pages/index.adoc:617]: Will discuss in Monday's CAWG meeting. My inclination is to leave it as is.

**Pull request merged**